lineman/app/services/intercom_service.coffee

```diff
@@ -40,8 +40,10 @@ angular.module('loomioApp').factory 'IntercomService', ($rootScope, $http, AppCo
           created_at: group.createdAt
 
     contactUs: ->
-      return unless @available()
-      window.Intercom.public_api.showNewMessage()
+      if @available()
+        window.Intercom.public_api.showNewMessage()
```

Comment:
We'll want to inject $window here, so that if we decide to write tests for this, we can control what's on the window object.

Annoying I know, but we've been using $window everywhere else and I don't want to start changing now.